doc-content/enterprise-only/vehicle-route-planning/vrp-auto-restart-con.adoc

```diff
@@ -1,13 +1,7 @@
 [id='vrp-auto-restart-con_{context}']
 
-= Spring Boot automatic restart
-https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-devtools-restart[Automatic restart]
-is provided by Spring Boot DevTools. When you run the OptaWeb Vehicle Routing back end with the Spring Boot Maven plug-in, the application automatically restarts whenever files on the classpath change.
-Automatic restart scans files on the classpath, so you only need to recompile your changes to trigger application restart.
-No IDE configuration is needed.
+= Quarkus development mode
 
-If your IDE has a compile-on-save feature (for example Eclipse or NetBeans), you just need to save the files that have changed since the last compilation.
+In development mode, the backend automatically restarts whenever you refresh the browser tab where the front end runs if there are any changes in the back-end source code or configuration.
```

Comment:
[translatability] wdyt?

s/In development mode, the backend automatically restarts whenever you refresh the browser tab where the front end runs if there are any changes in the back-end source code or configuration./In development mode, if there are changes to the back-end source code or configuration and you refresh the browser tab where the front end runs, the back-end automatically restarts.

Reply:
I like it.